cmake/BuildFlags.cmake

```diff
@@ -133,7 +133,9 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "MSVC"
   endif()
   set(TORCH_XPU_ARCH_LIST ${AOT_TARGETS} PARENT_SCOPE)
 
-  set(SYCL_OFFLINE_COMPILER_AOT_OPTIONS "-device ${AOT_TARGETS}")
+  if(AOT_TARGETS)
```

Comment:
Please follow TORCH_CUDA_ARCH_LIST usage in PyTorch. We behave like what CUDA behaves.